### DISCUSSION TITLE
GROWI Docsの「定期バックアップ」にリージョン名の記載を追加してはどうか

### DISCUSSION THREAD

**Discussion by minadzki** [growilabs/growi#9151] [Ideas / Requests]
現在、Cloudflare R2をオブジェクトストレージとしてGROWI 7.0.19を利用しています。R2ではリージョンがenum、wnum、eeur、weur、apacの5種類（プラスしてリージョンを特定しないautoがあります）であり、awscliのリージョン名バリデーションとの互換性を持つためにus-east-1がautoへのエイリアスとなっています。
mongodb-awesome-backup 0.6.3現在、Dockerfileにap-northeast-1をデフォルトリージョンとする記載があるため、GROWI Docsの記述やmongodb-awesome-backupのReadme.mdに単純に従って設定を行っただけでは、バックアップのアップロード時にリージョンエラーとなります。試していませんが、ほかのオブジェクトストレージでも起こりうるエラーかなと思います。

解決策としては、docker-compose.override.ymlに`- AWSCLIOPT=--region us-east-1`の1行を追加すればいいのですが、例えば以下のようにGROWI Docsの「MongoDBによる定期バックアップ#docker-composeによる管理」に記載しておくと親切でよいかなと思いました。

`- AWSCLIOPT=--region ${Your Region Code}`

また、mongodb-awesome-backupのReadme.mdについても、Usageに記載してもいいのではないかなと思います。Environment variablesには既に記載がありますが。

**Comment by yuki-takei** [comment]
@minadzki ご指摘ありがとうございます。ご提案の通り追記すべきかと思いますが、PR を送っていただくことは可能でしょうか？

ソースはこちらになります。
https://github.com/growilabs/growi-docs/edit/master/docs/ja/admin-guide/admin-cookbook/mongodb-backup-regular.md

お手隙の際には是非よろしくお願いいたします。
こちらでも手が空き次第追記します。
